birthday/birthday.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+"""birthday.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1_qsT3etlfX7XkZc6jXFyL_RE2sh_GrJ8
+"""
+
+import datetime as dt
+
+def calculate_age(birthdate):
+  today = dt.date.today()
+  age = today.year - birthdate.year
```

Comment:
This will give the correct answer but _only_ if the person's birthday has already happened this year. Can you think of a way to fix that?